h/config.py

```diff
@@ -56,9 +56,7 @@ def configure(environ=None, settings=None):  # pylint: disable=too-many-statemen
     settings_manager.set("mail.default_sender", "MAIL_DEFAULT_SENDER")
     settings_manager.set("mail.host", "MAIL_HOST")
     settings_manager.set("mail.port", "MAIL_PORT", type_=int)
-    settings_manager.set(
-        "sqlalchemy.url", "DATABASE_URL", type_=database_url, required=True
-    )
+    settings_manager.set("sqlalchemy.url", "DATABASE_URL", required=True)
```

Comment:
I removed the `database_url()` validation function because it seems to be unnecessary.